/**:
  ros__parameters:
    use_sim_time: True
    engine:
      foot_apex_phase: 0.5
      foot_z_pause: 0.0
      trunk_pause: 0.0
      foot_put_down_phase: 1.0

      kick_length: 0.09
      kick_vel: 0.2
      kick_phase: 0.75

    node:
      engine_freq: 200.0

      ik:
        timeout: 0.01
        reset: False

      debug_active: False

      max_step_x: 1000.0
      max_step_y: 1000.0
      max_step_xy: 1000.0
      max_step_z: 1000.0
      max_step_angular: 1000.0

      x_speed_multiplier: 1.0
      y_speed_multiplier: 1.0
      yaw_speed_multiplier: 1.0

      stability_stop:
        imu:
          active: False
          pitch:
            threshold: 0.19
            vel_threshold: 1.3
          roll:
            threshold: 0.4
            vel_threshold: 5.7
        pause_duration: 3.0

      imu_y_acc_tau: 0.1

      phase_reset:
        min_phase: 0.0
        foot_pressure:
          active: False
          ground_min_pressure: 0.0
        effort:
          active: False
          joint_min_effort: 0.0
        imu:
          active: False
          y_acceleration_threshold: 2.0

      trunk_pid:
        pitch:
          p: 0.0
          i: 0.0
          d: 0.0
          i_clamp_min: 0.0
          i_clamp_max: 0.0
          antiwindup: False
        roll:
          p: 0.0
          i: 0.0
          d: 0.0
          i_clamp_min: 0.0
          i_clamp_max: 0.0
          antiwindup: False
